testpage.1

```diff
@@ -41,9 +62,15 @@ name in the generated link will be prefixed by the dir given with
 This works for references specified as \fImanpage(3)\fP (within the emhpasis
 foformatting) and in a plain \.BR section (often used in the SEE ALSO
 section).
+
 .IP \-\-hrefdir=<dir>
 Specify a directory to prefix generated href links created with the \-\-mandir
 option. This defaults to ".".
+
+.IP "-?, --special-char"
+Options with special characters will be included in anchors.
+Such as this option with special character \fI\-?, \-\-special\-char\fP.
```

Comment:
I don't see any code handling this new option. Plus, I don't think this needs to be managed by an option - I think it makes sense to improve the general handling of anchors and links so it should probably be used by default.

Reply:
For the test build. Moved into description. Anyway options are named wrong (*--mumbo and --jumbo*), which gave me impression that it was only for build test purposes. Does not seem like an issue though (*maybe good for someone to find who has never contributed on github also*) .